tests/pytests/test_resp3.py

```diff
@@ -708,3 +708,374 @@ def test_profile_child_itrerators_array():
     }
     if not env.isCluster:  # on cluster, lack of crash is enough
         env.assertEqual(res, exp)
+
+def testExpandErrorsResp3():
+  env = Env(protocol=3)
+  # On JSON
+  env.cmd('ft.create', 'idx', 'on', 'json', 'SCHEMA', '$.arr', 'as', 'arr', 'numeric')
+  env.expect('FT.SEARCH', 'idx', '*', 'FORMAT').error()
+  env.expect('FT.SEARCH', 'idx', '*', 'FORMAT', 'XPAND').error()
+  env.expect('FT.AGGREGATE', 'idx', '*', 'FORMAT').error()
+  env.expect('FT.AGGREGATE', 'idx', '*', 'FORMAT', 'XPAND').error()
+
+  # On HASH
+  env.cmd('ft.create', 'idx2', 'on', 'hash', 'SCHEMA', '$.arr', 'as', 'arr', 'numeric')
+  if not env.isCluster():
+    env.expect('FT.AGGREGATE', 'idx2', '*', 'FORMAT', 'EXPAND').error()
+  else:
+    # TODO: Expect an error once MOD-5211 is done
+    env.expect('FT.AGGREGATE', 'idx2', '*', 'FORMAT', 'EXPAND').equal(
+       {'attributes': [], 'error': [], 'total_results': 0, 'format': 'EXPAND', 'results': []}
+    )
+
```

Comment:
why the difference?

Reply:
Coordinator doesn't return an error, just an empty reply (until MOD-5211 is fixed)